python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -553,13 +553,17 @@ def test_generic_hints(self):
     def test_extended_hint_types(self):
         df = self.spark.range(10e10).toDF("id")
         such_a_nice_list = ["itworks1", "itworks2", "itworks3"]
-        hinted_df = df.hint("my awesome hint", 1.2345, "what", such_a_nice_list)
+        hinted_df = df.hint("my awesome hint", 1.2345, "what", such_a_nice_list, 2)
         logical_plan = hinted_df._jdf.queryExecution().logical()
```

Comment:
Spark Connect does not have this `_jdf`. Let's either fix this test to don't use `_jdf` or, keep it skipped and add a new test at `test_connect_basic.py` for now.